src/app/templates/_.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - 5
```

Comment:
as mentioned above, we probably want `node` and `4.2` to get the latest supported stable version and latest LTS version.